agent/collector/ingester.go

```diff
@@ -28,13 +24,6 @@ func newForwardIngester(ctx context.Context, batchTimeout time.Duration, remoteI
 		traceCache:     remoteIngesterConfig.traceCache,
 	}
 
-	err := ingester.connectToRemoteServer(ctx)
```

Comment:
Having a connect here was failing the initialization and we are not connecting to a server "yet"

Reply:
We might need to disable all the removed code based on a flag. 

The removed code is important in the future when we decide to forward traces from the agent to an OTLP-compatible endpoint. But for now, I think we can remove it.